pandas/tests/indexes/interval/test_interval_ops.py

```diff
@@ -0,0 +1,15 @@
+"""Tests for Interval-Interval operations, such as overlaps, contains, etc."""
+import pytest
+
+from pandas import IntervalIndex
+from ...arrays.interval.interval_ops import BaseOverlaps
```

Comment:
use absolute imports. do we really need a mixin? (as opposed to fixtures)?

Reply:
Reverted back to using a fixture; the downside of using a fixture to test both is that you end up with `IntervalIndex` tests in the array subfolder which is intuitive.  The mixin admittedly adds some code bloat, so not really an ideal solution either.